### PR TITLE
Gkc javadoc v2

### LINKED ISSUE #180: JavaDoc all the GKC things
Bunch of stuff in [GKC](https://github.com/frc937/robot2024/tree/gkc-2024) needs JavaDoc. Keep in mind that commands only require JavaDoc for the classes themselves and their constructors.

- AddressableLightStrip.java:49: No description for `@param`
- AddressableLightStrip.java:50: No description for `@param`
- DriveToAmpBlue.java
- DrivetoAmpRed.java
- DriveToCenterAuto.java
- DumpNote.java
- OnePieceAutoButItWorksISwear.java
- PickUpFromCenterAuto.java
- RobotContainer.java:289: `@link` reference is misspelled
  - This is causing JavaDoc build to fail
- RobotContainer.java:275: JavaDoc public field
- RobotContainer.java:268: JavaDoc public field
- RobotContainer.java:273: JavaDoc public field
- RobotContainer.java:271: JavaDoc public field

### PR DESCRIPTION
**Pull request by Berdenson**
Closes #180 
Implements javadoc for all the gkc things

### CODE REVIEW

File: src/main/java/frc/robot/commands/auto/OnePieceAutoButItWorksISwear.java [hunk 1]
```diff
@@ -17,6 +17,8 @@
 // NOTE:  Consider using this command inline, rather than writing a subclass.  For more
 // information, see:
 // https://docs.wpilib.org/en/stable/docs/software/commandbased/convenience-features.html
+// One piece - the anime thing
+/** Auto that drives to the amp and offloads the note. */
```
Comment (quackitsquinn) [inline src/main/java/frc/robot/commands/auto/OnePieceAutoButItWorksISwear.java:21]: Add reasoning why this exists (non-functional limelight at comp)

**Review by quackitsquinn — CHANGES_REQUESTED** [review]
have fun :)

**Review by willitcode — CHANGES_REQUESTED** [review]
Please do the bare minimum and run spotless so that your code builds before requesting my review.

**Review by quackitsquinn — APPROVED** [review]

**Review by willitcode — APPROVED** [review]
eh probably good enough, I trust you to be more nitpicky about javadoc than me

**Pull request merged**